### ISSUE TITLE
NTR: FALDH activity low in leukocytes or fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8204]
New term request FALDH activity low in leukocytes or fibroblasts (segal_080125180818): Fatty aldehyde dehydrogenase

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000720
- label: Reduced tissue fatty aldehyde dehydrogenase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:1939650
- def: Activity of fatty aldehyde dehydrogenase (EC 1.2.1.3) in tissues below the lower limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts and leukocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8204
- synonyms:
